.github/workflows/packages_creation.yml

```diff
@@ -97,14 +108,19 @@ jobs:
       shell: bash
       run: echo ::set-output name=UPLOAD_URL::$( cat UPLOAD_URL.txt )
 
-    - name: Upload Release Asset
-      id: upload-release-asset
+    - name: Upload Release Asset icub-common
       uses: shogo82148/actions-upload-release-asset@v1
       with:
         upload_url: ${{ steps.read_upload_url.outputs.UPLOAD_URL }}
-        asset_path: ${{ steps.package_generation.outputs.ICUB_PACKAGE_NAME }}
+        asset_path: ${{ steps.package_generation_icub-common.outputs.ICUB_COMMON_PACKAGE_NAME }}
         asset_content_type: application/vnd.debian.binary-package
 
+    - name: Upload Release Asset icub-main
+      uses: shogo82148/actions-upload-release-asset@v1
+      with:
+        upload_url: ${{ steps.read_upload_url.outputs.UPLOAD_URL }}
+        asset_path: ${{ steps.package_generation_icub-main.outputs.ICUB_MAIN_PACKAGE_NAME }}
+        asset_content_type: application/vnd.debian.binary-package
```

Comment:
Please, let's use more standard actions as `actions/upload-release-asset`.

Take a look at https://github.com/icub-tech-iit/ventilator-FI5/blob/master/.github/workflows/create-release.yml for an example.